github/github.go

```diff
@@ -154,6 +156,19 @@ type UploadOptions struct {
 	Name string `url:"name,omitempty"`
 }
 
+// MediaRawType represents raw return of a request instead of json format
+type MediaRawType int
+
+const (
+	Diff MediaRawType = 1 + iota
+	Patch
+)
+
+// GetRawOptions specifies the optional parameter when user want to get raw presentation of a response instead of json format
+type GetRawOptions struct {
```

Comment:
What do you think about shortening the name to `RawOptions`?

If you look at other structs in this package that have `Options` suffix, they're usually named after the general area of interest, not the exact method name.